docs/manual/guieffect-checker.tex

```diff
@@ -277,6 +277,10 @@ \subsection{Using an effect-polymorphic type\label{guieffect-polymorphism-using}
 
 It is an error to apply an effect instantiation qualifier to a type that is not effect-polymorphic.
 
+Note that no annotation is required on the anonymous class declaration itself (e.g. \code{new Runnable()\{...\}} 
+does not require a type use annotation, although the variable, field, or argument it ends up being assigned to might).
+Instead, the GUI Effect Checker will infer the effect qualifier based on the method being called from within the 
+members of that specific anonymous class.
```

Comment:
To make things explicit, here is a note on the type use annotation being no longer needed. Let me know if there is anything else missing, @csgordon? (Also, might need to validate the LaTeX still, what is the target for building the manual, again? 😅 )